executor/builder.go

```diff
@@ -611,21 +613,21 @@ func (b *executorBuilder) buildSemiJoin(v *plan.PhysicalHashSemiJoin) *HashSemiJ
 	return e
 }
 
-func (b *executorBuilder) buildAggregation(v *plan.PhysicalAggregation) Executor {
-	if v.AggType == plan.StreamedAgg {
-		return &StreamAggExec{
-			baseExecutor: newBaseExecutor(v.Schema(), b.ctx, b.build(v.Children()[0])),
-			StmtCtx:      b.ctx.GetSessionVars().StmtCtx,
-			AggFuncs:     v.AggFuncs,
-			GroupByItems: v.GroupByItems,
-		}
-	}
+func (b *executorBuilder) buildHashAgg(v *plan.PhysicalHashAgg) Executor {
 	return &HashAggExec{
 		baseExecutor: newBaseExecutor(v.Schema(), b.ctx, b.build(v.Children()[0])),
 		sc:           b.ctx.GetSessionVars().StmtCtx,
 		AggFuncs:     v.AggFuncs,
 		GroupByItems: v.GroupByItems,
-		aggType:      v.AggType,
```

Comment:
Can we remove the field of `aggType`？

Reply:
We use different structs for stream and hash agg, so aggType is useless.